Assets/Scripts/Robot/ApplicationAPIVersion.cs

```diff
@@ -11,28 +11,78 @@ public class ApplicationAPIVersion : Singleton<ApplicationAPIVersion>
     {
         private string appVersion;
 
+        private string line;
+
         void Start()
         {
             #if UNITY_EDITOR_WIN
                 string path = "Assets/Scripts/reachy2-sdk-api/python/reachy2_sdk_api/__init__.py";
+                if (!File.Exists(path))
+                {
+                    Debug.LogError($"Version file not found at path: {path}");
+                    return;
+                }
+
+                try
+                {
+                    bool versionFound = false;
```

Comment:
the parsing of the py file could be in a function. The logic would be easier to understand